internal/pay/data.go

```diff
@@ -24,7 +24,7 @@ type Link struct {
 }
 
 type CreatePaymentResponse struct {
-	CreatedDate     time.Time       `json:"created_date"`
+	CreatedDate     date.Date       `json:"created_date"`
```

Comment:
![image](https://github.com/user-attachments/assets/7629e94e-859b-48e0-b23e-c10d26523c46)

docs suggest `time.Time` is correct here